setup.py

```diff
@@ -43,6 +43,7 @@
         'pyyaml',
         'requests',
         'scitools-iris>=2.2',
+        'requests',
```

Comment:
If you merge the latest master branch into this branch, this should already be included since #850